### PR TITLE
make call createMilestoneComment on newIssue func

### LINKED ISSUE #8644: New Issue Missing Milestone Comment
gitea does not create milestone comment on new issue
I also tried it on try.gitea.io

https://try.gitea.io/jaqra/Foo/issues/4

### PR DESCRIPTION
**Pull request by ghost**
fixes #8644

### CODE REVIEW

**Review by guillep2k — APPROVED** [review]

File: models/issue.go [hunk 1]
```diff
@@ -970,6 +970,10 @@ func newIssue(e *xorm.Session, doer *User, opts NewIssueOptions) (err error) {
 		if _, err = e.Exec("UPDATE `milestone` SET num_issues=num_issues+1 WHERE id=?", opts.Issue.MilestoneID); err != nil {
 			return err
 		}
+
+		if _, err = createMilestoneComment(e, doer, opts.Repo, opts.Issue, -1, opts.Issue.MilestoneID); err != nil {
```
Comment (guillep2k) [inline models/issue.go]: Sorry, I've seen this only after I approved. The value for `oldMilestoneID` must be 0, not -1.

Comment (jaqra) [reply models/issue.go]: I made -1 too because old code was

https://github.com/go-gitea/gitea/pull/8221/files#diff-d5eea3b2c0d4c92d62e12b34eb8da2c0L1122

Comment (guillep2k) [reply models/issue.go]: I see. But when you assign an existing issue to a milestone, the number used for that column is 0. That's an inconsistency.

Comment (jaqra) [reply models/issue.go]: done

**Comment by lunny** [comment]
Thanks! @jaqra

**Review by lunny — APPROVED** [review]

**Pull request merged**